core/tests/puppeteer-acceptance-tests/utilities/user/logged-in-user.ts

```diff
@@ -76,6 +76,53 @@ export class LoggedInUser extends BaseUser {
       throw new Error(`Exploration with title ${title} is not present.`);
     }
   }
+
+  /**
+   * Navigates to the community library page.
+   */
+  async navigateToCommunitylibrary(): Promise<void> {
+      await this.page.goto(testConstants.URLs.CommunityLibrary, {
+        waitUntil: 'load',
+      });
+  }
+
+  /**
+   * Views all featured activities on the community library page.
+   */
+  async viewAllFeaturedActivities(): Promise<Array<object>> {
+      await this.page.waitForSelector('.oppia-library-group');
```

Comment:
Please always store the selector strings as constant to keep the code clean.

Reply:
done.